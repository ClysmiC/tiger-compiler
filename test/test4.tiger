let
        var _a : int;
in
        a := 1;
        printi(a);
end
