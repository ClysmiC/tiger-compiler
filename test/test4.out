LET VAR
Scanner error (line 2): let _<---
                        "_" does not begin a valid token.
COLON ID SEMI IN ID ASSIGN INTLIT SEMI ID LPAREN ID RPAREN SEMI END
unsuccessful parse
